test/sql/test_alter_table/T/test_alter_table_abnormal

```diff
@@ -0,0 +1,98 @@
+-- name: test_alter_pk_table_abnormal
+
+CREATE TABLE t0(k0 BIGINT, k1 DATETIME, v0 INT, v1 VARCHAR(100))
+ primary key(k0, k1)
+ distributed by hash(k0)
+ buckets 1
+ order by (v0)
+ properties('replication_num'='1');
+
```

Comment:
Why not inserting some data into this table and add some select statements to test if the column is added/dropped correctly?

Reply:
I have added an `DESC <table>` statement in the tests to check the table structure.

Reply:
Got it, how about inserting some data?

Reply:
Updated